src/design_docs/org/lib/app.coffee

```diff
@@ -1,22 +1,38 @@
 _ = require('./underscore')
 h = require('./helpers')
-validation = require('./validation/validate')
+validate = require('./validation/validate')
 actions = require('./actions')
 audit = require('./shared/audit')
 
-dd =
-  views: {}
+auth = validate.auth
 
+dd =
+  views:
+    by_role:
+      map: (doc) ->
+        validate = require('views/lib/validation/validate')
+        if not validate._is_team(doc)
+          return
+        team_id = doc._id.slice(5)
```

Comment:
upon further reflection, this is a good catch. instead of using a sliced _id, should be using the name, which already has the type removed from the id. I've created an issue (#21) to track this. I think it should be ok to merge with this here.